module-utils/time/test/unittest_time.cpp

```diff
@@ -342,6 +346,55 @@ TEST_CASE("DateTime")
         }
     }
 
+    SECTION("isCurrentWeek")
+    {
+        auto newTimeTimeinfo = currentTimeTimeinfo;
+        SECTION("current week")
+        {
+            auto i = GENERATE(range(2, 7));
+            newTimeTimeinfo.tm_mday -= i;
+            const auto newTime = std::mktime(&newTimeTimeinfo);
+            DateTime datetime(timeSettings, newTime, currentTime);
+            REQUIRE(datetime.isCurrentWeek());
+        }
+        SECTION("not current week")
+        {
+            auto newTimeTimeinfo = currentTimeTimeinfo;
+            SECTION("different day")
+            {
+                auto i = GENERATE(range(7, 32));
+                newTimeTimeinfo.tm_mday -= i;
+                auto newTime = std::mktime(&newTimeTimeinfo);
+                DateTime datetime(timeSettings, newTime, currentTime);
+                REQUIRE(!datetime.isCurrentWeek());
+            }
+            SECTION("different month")
```

Comment:
Test names like this don't tell much about what's the test supposed to do :( I can see here we have generating months January-December and... what happens next? I mean the test name should tell what's checked without diving into the code under test

Reply:
If you look at the whole test, can't you see that we change the month in the date to test whether the isCurrentWeek function works correctly?
```
TEST_CASE("DateTime")
...
	SECTION("isCurrentWeek")
	...
		SECTION("not current week")
		...
			SECTION("different month")
			...
```

Reply:
No :D Anyway, I won't insist - leave it as is if you prefer

Reply:
Fixed :)